text/0005-global-init.md

```diff
@@ -51,10 +51,12 @@ party libraries to use the global SDK before it is installed, which is
 addressed in a requirement stated below.
 
 The explicit initializer method should take independent `Tracer` and
-`Meter` objects (e.g., `opentelemetry.Init(Tracer, Meter)`).  The SDK
-may be installed no more than once.  After the first SDK installed,
-subsequent calls to the explicit initializer shall log console
-warnings.
+`Meter` factories (e.g., `opentelemetry.Init(TracerFactory,
+MeterFactory)`), factories because they facility explicitly named
```

Comment:
this line/sentence seems broken: "they facility explicitly"?

Reply:
Fixed, should have been "they construct explicitly named ..."